src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -384,6 +390,7 @@ private static Exception getExceptionFromJson(String exceptionJson) {
             }
             return getObjectMapper().readValue(exceptionJson, Exception.class);
         } catch (IOException ex) {
+            logger.error("JACKSON deserialization failed. JSON:" + exceptionJson);
```

Comment:
```suggestion
            logger.error("JACKSON deserialization failed. JSON:" + exceptionJson, ex);
```

If you pass in `ex` as a second positional argument, it will print the stack trace in the log :)

Reply:
Is that always true? I have had difficulty finding stack trace in the stackdriver logs, but that may just be some lack of skill on my part.

Reply:
The ex variable here is the JACKSON failure to deserialize JSON exception, not the actual underlying cause -- that's in the exceptionJson variable. The throw on the next line includes the JACKSON exception. That's the reason I didn't include ex in the logger call -- because I *thought* it would be redundant. But maybe it's good to keep it together or I'm mistaken about the exception eventually being logged higher up? What do you think?

Reply:
I'd prefer to err on the side of more info than less in the logs at least for now, since we are missing stacktraces in a few places. @ddietterich yes this is always the case for `logger.error`. I am unsure if it works for the other log levels but I would assume so.